src/celeritas/ext/GeantPhysicsOptions.hh

```diff
@@ -86,6 +86,17 @@ struct GeantPhysicsOptions
     RelaxationSelection relaxation{RelaxationSelection::none};
     //!@}
 
+    //!@{
+    //! \name Optical photon physics
+    //! Enable Cerenkov physics
+    bool cerenkov{true};
+    //! Enable scintillation
+    bool scintillation{true};
+    //! Enable optical Rayleigh scattering
+    bool optical_rayleigh{true};
```

Comment:
Do these do anything? I don't see any changes in GeantImporter itself.

Reply:
They are used in `Runner::build_optical_collector`. Though `optical_rayleigh` is not used yet (the optical collector input does not load Rayleigh data).